packages/components/inputs/select-input/src/select-input.spec.js

```diff
@@ -77,6 +77,26 @@ it('should have focus automatically when isAutofocussed is passed', () => {
   expect(getByLabelText('Fruit')).toHaveFocus();
 });
 
+it('should have a open menu if menuIsOpen is true', () => {
+  const { getByLabelText, getByText } = renderInput({
+    menuIsOpen: true,
+  });
+  const input = getByLabelText('Fruit');
+
+  expect(getByText('Mango')).toBeInTheDocument();
+});
+
+it('should not have a open menu if menuIsOpen is true and isReadOnly is true', () => {
```

Comment:
NIT 
```suggestion
it('should not have an open menu if menuIsOpen is true and isReadOnly is true', () => {
```

This also applies to the other places in the code.

Reply:
Good catch